cmd/demo/demo.go

```diff
@@ -83,6 +83,7 @@ func render(rsp interface{}, table *tablewriter.Table) bool {
 
 func main() {
 	ds := flag.String("datasource", "", "database connect string")
+	modelDir := flag.String("model_dir", "", "model would be saved on the local dir, otherwise upload to the table.")
```

Comment:
"save the model to SQL engine as a human-unreadable table if this option is empty"